cmake/modules/FindDEAL_II_KOKKOS.cmake

```diff
@@ -162,4 +165,34 @@ if(KOKKOS_FOUND)
     # warning #940-D: missing return statement at end of non-void function
     enable_if_supported(DEAL_II_CXX_FLAGS "-Xcudafe --diag_suppress=940")
   endif()
+
+  #
+  # Extract version numbers:
+  #
+  if(NOT KOKKOS_VERSION)
+    if(Kokkos_VERSION)
+      set(KOKKOS_VERSION "${Kokkos_VERSION}")
+    else()
+      message(FATAL_ERROR "FindPackage(Kokkos) did not set KOKKOS_VERSION!")
```

Comment:
I think @tamiko did not approve of this

Reply:
I removed this part on @tamiko 's suggestion, but the origin of this was different from the case you have in mind: You were calling `FATAL_ERROR` in a place where a macro fails because it couldn't find Kokkos, when there was a back-up search that would run later. Here, I'm simply asserting that whatever found Kokkos somewhere has correctly set the version number. It would be a bug in our cmake machinery if it didn't.

Either way, the issue has become moot.